hotel_management/settings.py

```diff
@@ -131,6 +132,18 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'static/images')
 STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
 
+# Emailing settings
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

Comment:
Đẩy toàn bộ env vào .env và đọc từ đó.
git ignore .env.
Thêm một file .env.example với nội dung kiểu: 
EMAIL_FROM = XXXX
EMAIL_HOST_USER = XXXX
EMAIL_HOST_PASSWORD = XXXX
Với nội dung các key đc masking.

Reply:
Done